### PR TITLE
Fixes /me event names

### PR DESCRIPTION
**Pull request by oguzkocer**
Fixes the Nosara event names for /me so it follows the "action at the end" convention. /cc @astralbodies 

Needs review: @daniloercoli

### CODE REVIEW

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java [hunk 1]
```diff
@@ -273,13 +273,13 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
                 predefinedEventProperties.put("menu_item", "site_settings");
                 break;
             case OPENED_ACCOUNT_SETTINGS:
-                eventName = "me_opened_account_settings";
+                eventName = "me_account_settings_opened";
```
Comment (astralbodies) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: Can we drop the `me_` prefix on both of these events? There has been some history already on the iOS side with `account_settings_opened` so I don't want to affect any funnels already using that. Although I can be compelled to add `me_` to iOS if @daniloercoli feels strongly about it.

Comment (oguzkocer) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: Sure I can. The reason I've added the `me_` prefix was most of the other events have that kind of prefix to identify where the event is related to. I guess these ones are generic enough that we don't need. Will wait for confirmation from @daniloercoli and drop it if he agrees. Thanks!

Comment (daniloercoli) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: I don't have opinions on that. Drop them.

Comment (oguzkocer) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: Done!

**Comment by daniloercoli** [comment]
:shipit:

**Pull request merged**